### DISCUSSION TITLE
The Datadog integration is working great, but how can we push historical sentry logs?

### DISCUSSION THREAD

**Discussion by henry-mcdonald** [getsentry/support#27] [Support] [answered]
We are piping Sentry logs into Datadog.

The integration works great for logs that are produced after the integration was complete. 

Is there a way we can push the historical logs into Datadog?

**Comment by kerenkhatiwada** [comment, marked as answer]
It is not currently possible to push the historical logs into Datadog. Please see Sentry's data and visualization integrations [here.](https://docs.sentry.io/product/integrations/data-visualization/)
